misc/testboard/testboardpush

```diff
@@ -0,0 +1,307 @@
+            subrepo_default = manifest_spec.find("default")
+            subrepo_default = subrepo_default if subrepo_default is not None else manifest_spec
+            for subrepo in man_subrepos:
+                # Subprocess output: "<hash> <one-line commit message> [<relative committer date>]"
+                subrepo_name = subrepo.get("name")
+                subrepo_path = subrepo.get("path") or subrepo_name
+                subrepo_rev  = subrepo.get("revision") or subrepo_default.get("revision") or 'UNKNOWN'
+                try:
+                    subrepo_rev_log = git_in_subrepo(subrepo_path, ['log', subrepo_rev, '-n', '1', '--format=%h %s [%cr]'])
+                except subprocess.CalledProcessError as e:
+                    log("Seems like the reference %s is not in %s" % (subrepo_rev, subrepo_path))
+                    debug(e.output)
+
+                commit_template += '{:{}} {}'.format('[' + subrepo_name + ']', subrepo_column_width + 2, subrepo_rev_log)
+
+            commit_template += '\n'
+            for subrepo in subrepos:
+                commit_template += "{0:{1}} {2[0]}/{2[1]}\n".format(subrepo + ':', subrepo_column_width + 1, remote_url[subrepo])
+
+            # L4V_ARCH tags
+            l4v_archs = ['ARM','ARM_HYP','RISCV64','X64']
+            arch_line = 'L4V_TEST_ARCHS: ' + ' '.join(['[%s]' % arch for arch in l4v_archs])
+
+            git_template = f"""{commit_template}
+
+# Remove from L4V_TEST_ARCHS any architecture you don't want to be tested
+{arch_line}
+
+# Enter commit message. Lines beginning with # are ignored.
+
+"""
+
+            git_template_file = os.path.join(testboard_repo, '.gittemplate')
+            with open(git_template_file, 'a') as f:
+                f.write('\n' + git_template)
+            try:
+                set_template = subprocess_output(['git', 'config', 'commit.template', '.gittemplate'], cwd=testboard_repo)
+            except subprocess.CalledProcessError as e:
+                fatal("Failed to set git template. Log:\n" + e.output)
+
+            commit_command = ['git', 'commit', 'manifest.xml']
+        else:
+            commit_command = ['git', 'commit', 'manifest.xml', '-m', args.message]
+
+        try:
+            subprocess.check_call(commit_command, cwd=testboard_repo)
+        except subprocess.CalledProcessError:
+            fatal("Failed to commit to the test board repository.")
+
+        # Now attempt to push it.
+        # This may fail if someone else has pushed a commit in the meantime.
+        try:
+            subprocess.check_call(['git', 'push', '--set-upstream', 'origin', args.testboard_branch], cwd=testboard_repo)
+        except subprocess.CalledProcessError:
+            fatal("Failed to push to test board.")
+
+        testboard_oneline = subprocess_output(['git', 'log', '--oneline'], cwd=testboard_repo).strip()
+        log("Done. Your test board commit is:\n  " + testboard_oneline)
+    finally:
+          shutil.rmtree(testboard_repo)
```

Comment:
Nit: indentation.

```suggestion
        shutil.rmtree(testboard_repo)
```

Reply:
Interesting that the style check didn't catch this! Will have a look.

Reply:
I only noticed it when I diffed against the old script in l4v-internal.

Reply:
It was probably tabs vs spaces or something like that (the style checker accepts both version). I had commented out the `rmtree` for testing and who knows what fancy things my editor did that ;-). 

Will force-push the tweak.